modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1194 @@
+
+To add a `sidebar_show: '<sidebar-name>'` to `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  sidebar_show: '<sidebar-name>',
+});
+----
+
+If the sidebar is custom, the sidebar should be setup in the `setup` section of `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea', // change this value according to your HTML
+  setup: (editor) => {
+    editor.ui.registry.addSidebar('<sidebar-name>', {
+      tooltip: '<sidebar-name>',
+      icon: '<icon-name>',
+      onSetup: (api) => {
+        console.log('Render panel', api.element());
+      },
+      onShow: (api) => {
+        console.log('Show panel', api.element());
+      },
+      onHide: (api) => {
+        console.log('Hide panel', api.element());
+      }
+    });
+  }
+});
+----
+
+NOTE: The sidebar presents with no animation or other effect. It is just there, along with any other initialised {productname} editor components. Also, as noted above, the sidebar does not take focus, which remains with the main editor.
+
+////
+A common envisioned use is to initialise a {productname} editor with the Premium Plugin Comments sidebar automatically present.
+
+After adding the Comments Premium Plugin, to add a Comments sidebar, add `sidebar_show: 'showcomments'` to `tinymce.init`:
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea", // change this value according to your HTML
+  sidebar_show: 'showcomments',
+});
+----
+////
+
+
+=== New `newline_behavior` option controls what happens when the Return or Enter key is pressed or the `mceInsertNewLine` command is used
+
+Setting `forced_root_block` to `false` or an empty string was deprecated in {productname} 6.0. This option now only accepts valid block elements.
+
+However, some use cases relied on `forced_root_block: false` to keep material created in {productname} for use and presentation elsewhere from being placed inside the `<p>` block element.
+
+The `newline_behavior` option has been added to {productname} 6.1 to cater to these use-cases.
+
+`newline_behavior` affects what is added to a document when the `mceInsertNewLine` command is run or when any of the four _create a new line_ keystrokes are pressed: *Return*, *Enter*, *Shift-Return*, or *Shift-Enter*.
+
+By default these keystrokes work as follows:
+
+1. pressing *Return* or *Enter* creates a new paragraph.
++
+For example, in ordinary text, pressing *Return* closes the current `<p>` tag, moves the insertion point in the {productname} editor to a new line and opens a new `<p>` tag.
+   
+2. pressing *Shift+Return* or *Shift+Enter* creates a new line.
++
+For example, in ordinary text, pressing *Shift+Return* enters two `<br>` tags and moves the insertion point in the {productname} editor to a new line.
+
+The `newline_behavior` option takes one of four values:
+
+[cols="1,1"]
```

Comment:
```suggestion
[%header, cols="1,1"]
```
This is missing the markup to make the first row a table header. IIRC this is the correct shorthand markup.